projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -51,7 +51,7 @@ protected function __construct() {
 		add_action( 'admin_menu', array( $this, 'hide_parent_of_hidden_submenus' ), 99999 );
 
 		add_filter( 'admin_menu', array( $this, 'override_svg_icons' ), 99999 );
-		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ), 11 );
```

Comment:
It seems like `11` is too specific, if we want these scripts to always load last should we go for `99999`?

Reply:
We can do that although the default value is `10` which is what it has been at without problem until now. Bumping it to `11` fixes this particular issue though, I tried to avoid bumping it up so much just incase there was a reason we left it at `10` previously.

Reply:
I wrote the initial code there, and can't remember any reason ;-)

Reply:
Ha, well seeing as though it's only 1 above the default (what it was) and it's not been problematic up until now, shall we keep it at `11` and if there is any future issues increase it more significantly?

Reply:
Ok, sounds like a plan!